commands/login.go

```diff
@@ -124,8 +141,12 @@ func promptAuthenticationMethod() {
 		Label: "Select authentication method",
 		Items: []string{
 			"Email",
+			//colorizer.Sprintf(
+			//	"Authentication token (can be found under %s)",
+			//	colorizer.Bold(colorizer.Green("https://dashboard.tenderly.co/account/authorization")),
+			//),
 			colorizer.Sprintf(
-				"Authentication token (can be found under %s)",
+				"Access key can be generated on %s",
```

Comment:
My mistake on the wording, it should be:
```suggestion
				"Access key can be generated at %s",
```